contracts/sfc/ConstantsManager.sol

```diff
@@ -48,7 +48,7 @@ contract ConstantsManager is Ownable {
     error ValueTooLarge();
 
     function initialize() external initializer {
-        Ownable.initialize(msg.sender);
+        __Ownable_init(msg.sender);
```

Comment:
Optional: As initialize() is used only after constructing the ConstantsManager inside of NetworkInitializer, we may change this into a constructor.
If you do, please specify the owner as a parameter - using msg.sender in constructor would be considered an audit issue.